src/test/resources/alma-fix/99370771475306441.json

```diff
@@ -245,6 +245,13 @@
       "id" : "https://d-nb.info/gnd/4181939-1",
       "gndIdentifier" : "4181939-1"
     } ]
+  }, {
+    "notation" : "MS 6020",
```

Comment:
Seems correct.

Reply:
With my last adjustment of the regex this is dropped now. Since the signature is not validly build after RVK Signatur rules: https://rvk.uni-regensburg.de/images/stories/fruit/sigreg/regeln_fr_die_signaturvergabe_version_2012_02_17.pdf